pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -263,6 +263,9 @@ func generateCloudConfig(configFile *ConfigFile) (cloudConfig *CloudConfig, err
 			glog.Errorf("Encountered error for creating alpha feature gate: %v", err)
 		}
 		cloudConfig.AlphaFeatureGate = alphaFeatureGate
+	} else {
+		featureMap := make(map[string]bool)
```

Comment:
Actually I would like to move these to [Line 242](https://github.com/kubernetes/kubernetes/blob/e93819049db49694718bc9c96e67050d366c6f63/pkg/cloudprovider/providers/gce/gce.go#L242).